{
  Throwable t=extractException(runnable);
  assertException(exceptionClass,t);
}
